addons/languages/english/pack/src/main/res/values/english_pack_strings.xml

```diff
@@ -12,4 +12,5 @@
     <string name="workman_keyboard_description">Layout by OJ Bucao</string>
     <string name="halmak_keyboard_description">Nikolay Nemshilov\'s Halmak layout, implemented by Zorkedon</string>
     <string name="english_keyboard_qwertz_symbols_description">QWERTZ Latin keyboard with symbols, by LuKe</string>
+    <string name="swiftkey_keyboard_description">SwiftKey-style layout by Luc Street</string>
```

Comment:
Swiftkey is a trademark of Microsoft: https://trademarks.justia.com/857/78/swiftkey-85778862.html

Can you find a different wording here?

Reply:
@menny Renaming all instances of `swiftkey` to `swiftkey_style` or `swiftkey_like` sound ok? Or should I remove the term from this PR completely? I saw trademarks like "iPhone" elsewhere in the code so I wasn't sure

Reply:
I would suggest rename `swiftkey` to `minimal_qwerty`